articles/native-platforms/windowsphone/01-login.md

```diff
@@ -6,9 +6,11 @@ description: This tutorial will show you how to use the Auth0 Windows Phone SDK
 ## Windows Phone Tutorial
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'Auth0.WindowsPhone',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-WindowsPhone-samples',
+  githubUrl:'https://github.com/auth0-samples/auth0-WindowsPhone-samples/tree/master/00-Starter-Seed',
```

Comment:
@mehreencs87 Please update the URL to `https://github.com/auth0-samples/auth0-WindowsPhone-samples/tree/master/00-Starter-Seed/WindowsPhoneSilverlight` so that it matches the downloaded seed folder.